helpers/command.py

```diff
@@ -495,3 +507,58 @@ def version(cls):
             Config.KOBO_INSTALL_VERSION,
             stdout.strip()[0:7],
         ), CLI.COLOR_SUCCESS)
+
+    @staticmethod
+    def __validate_custom_yml(config, command):
+        """
+        Validate whether docker-compose must starts the containers with a user's
+        custom YML file in addition to already existing ones and especially if
+        this file already exists. If it does not, kobo-install is paused until
+        the user resumes it manually.
+
+        If user has chosen to use a custom YML file, it is injected in `command`
+        before being sent to shell.
+        """
+        dict_ = config.get_dict()
+        frontend_command = True
+        # Detect if it's a front-end command or back-end command
+        for part in command:
+            if 'backend' in part:
+                frontend_command = False
+                break
+
+        if frontend_command and dict_['use_frontend_custom_yml']:
+            custom_file = '{}/docker-compose.frontend.custom.yml'.format(
+                dict_['kobodocker_path']
+            )
+            if not os.path.exists(custom_file):
+                message = (
+                    'Please create your custom configuration in\n'
+                    '`{custom_file}`.'
+                ).format(custom_file=custom_file)
+                CLI.framed_print(message, color=CLI.COLOR_INFO, columns=75)
+                input('Press any key when it is done...')
+
+            # Add custom file to docker-compose command
+            command.insert(5, '-f')
+            command.insert(6, 'docker-compose.frontend.custom.yml')
+
+        if not frontend_command and dict_['use_backend_custom_yml']:
+            backend_server_role = dict_['backend_server_role']
+            custom_file = '{}/docker-compose.backend.{}.custom.yml'.format(
+                dict_['kobodocker_path'],
+                backend_server_role
+            )
+            if not os.path.exists(custom_file):
+                message = (
+                    'Please create your custom configuration in\n'
+                    '`{custom_file}`.'
+                ).format(custom_file=custom_file)
+                CLI.framed_print(message, color=CLI.COLOR_INFO, columns=75)
```

Comment:
Maybe we should increase the number of columns so it doesn't break onto the next line?
```
╔═════════════════════════════════════════════════════════════════════════╗
║                                                                         ║
║ Please create your custom configuration in                              ║
║ `/home/josh/kobo-tmp/kobo-docker/docker-                                ║
║ compose.backend.primary.custom.yml`.                                    ║
║                                                                         ║
╚═════════════════════════════════════════════════════════════════════════╝
```
Rather
```
╔═══════════════════════════════════════════════════════════════════════════════╗
║                                                                               ║
║ Please create your custom configuration in                                    ║
║ `/home/josh/kobo-tmp/kobo-docker/docker-compose.backend.primary.custom.yml`.  ║
║                                                                               ║
╚═══════════════════════════════════════════════════════════════════════════════╝
```